Detectors/MUON/MCH/Simulation/test/CMakeLists.txt

```diff
@@ -1,12 +1,41 @@
+#set(MODULE_NAME "MCHTest")
```

Comment:
for the final PR please cleanup (remove) the commented parts

Reply:
ok, done.